### PR TITLE
CUIRA-59

### PR DESCRIPTION
**Pull request by sonny-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CUIRA-59

### Change description ###
Initial project setup - ran the init.sh script to replace require properties

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -1,20 +0,0 @@
-name: CI Checks
-
-on:
```
Comment (kiran-yenigala-hmcts) [inline .github/workflows/checks.yml:3]: better to keep these

File: .github/workflows/codeql.yaml [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-      # If this step fails, then you should remove it and run the build manually (see below)
-      - name: Autobuild
-        uses: github/codeql-action/autobuild@v2
-
-      # ℹ️ Command-line programs to run using the OS shell.
-      # 📚 See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
-
-      #   If the Autobuild fails above, remove it and uncomment the following three lines.
-      #   modify them (or add more) to build your code if your project, please refer to the EXAMPLE below for guidance.
-
```
Comment (kiran-yenigala-hmcts) [inline .github/workflows/codeql.yaml:68]: better to keep these

File: docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   # TODO: change this name to match the name of your app
-  expressjs-template:
+  ra:
```
Comment (kiran-yenigala-hmcts) [inline docker-compose.yml:5]: is this ra or cui-ra?

**Review by kiran-yenigala-hmcts — COMMENTED** [review]
Minor Comments

File: docker-compose.yml (hunk 3 above)
Comment (sonny-hmcts) [reply docker-compose.yml:5]: The name was changed by the bin/init.sh script. Should i change this to cui-ra ?

File: .github/workflows/codeql.yaml (hunk 2 above)
Comment (sonny-hmcts) [reply .github/workflows/codeql.yaml:68]: These where removed by the bin/init.sh script. Should i put them back in ?

**Review by ben-james-hmcts — APPROVED** [review]
All looks good

**Pull request merged**